src/gui/newwizard/setupwizardcontroller.cpp

```diff
@@ -173,91 +183,122 @@ void SetupWizardController::nextStep(std::optional<PageIndex> currentPage, std::
     }
 
     if (desiredPage == 1) {
-        // first, we must resolve the actual server URL
-        auto resolveJob = Jobs::ResolveUrlJobFactory(_networkAccessManager).startJob(_accountBuilder.serverUrl());
-
-        connect(resolveJob, &CoreJob::finished, this, [this, resolveJob, showFirstPage]() {
-            resolveJob->deleteLater();
-
-            if (!resolveJob->success()) {
-                // resolving failed, we need to show an error message
-                showFirstPage(resolveJob->errorMessage());
-                return;
-            }
-
-            const auto resolvedUrl = qvariant_cast<QUrl>(resolveJob->result());
-
-            // next, we need to find out which kind of authentication page we have to present to the user
-            auto authTypeJob = DetermineAuthTypeJobFactory(_networkAccessManager).startJob(resolvedUrl);
+        auto *messageBox = new QMessageBox(
```

Comment:
hm, I can not figure how that is deleted later. `WA_DeleteOnClose` is AFAIK off by default, and there is no delete somewhere.

Reply:
Yes please add a WA_DeleteOnClose

Reply:
Fixed, thanks.